src/persist-client/src/read.rs

```diff
@@ -129,31 +124,49 @@ where
     /// The returned `Antichain` represents the subscription progress as it will
     /// be _after_ the returned parts are fetched.
     #[instrument(level = "debug", skip_all, fields(shard = %self.listen.handle.machine.shard_id()))]
-    pub async fn next(&mut self) -> (Vec<LeasedBatchPart<T>>, Antichain<T>) {
+    pub async fn next_parts(&mut self) -> Vec<ListenEvent<T, LeasedBatchPart<T>>> {
```

Comment:
I prefer the old names, please!

Reply:
Oh, I changed them to be consistent with the `ReadHandle` where `next()` gives you data and `next_parts()` gives you parts. Just making sure you're aware that they are different there

Reply:
As per slack discussion I updated the PR to make the method names consistent the other way around so now both `Listen` and `Subscribe` have `next` and `fetch_next` and they mean the same thing